src/components/common/Modal/AlertDialog.tsx

```diff
@@ -1,57 +1,73 @@
 import styled from '@emotion/styled';
 import { Common } from '@styles/globalStyle';
-import Button from '../Button/Button';
+import Button from '../Button';
+import { ReactNode } from 'react';
 
 interface Props {
-  content: string;
+  type?: 'alert' | 'complete' | 'warning';
+  content: string | ReactNode;
   onRequestClose: () => void;
+  onRequestConfirm: () => void;
 }
-const AlertDialog = ({ content, onRequestClose }: Props) => {
+const AlertDialog = ({
+  type = 'alert',
+  content,
+  onRequestClose,
+  onRequestConfirm,
+}: Props) => {
+  const btnBgColor = () => {
+    let btnBgColors = {
+      cancelBtn: Common.colors.primary05,
+      confirmBtn: Common.colors.primary,
+    };
+
+    if (type === 'complete') {
+      btnBgColors.cancelBtn = Common.colors.complete05;
+      btnBgColors.confirmBtn = Common.colors.complete;
+    } else if (type === 'warning') {
+      btnBgColors.cancelBtn = Common.colors.warning05;
+      btnBgColors.confirmBtn = Common.colors.warning;
+    }
+
+    return btnBgColors;
+  };
```

Comment:
`let`을 사용하는 대신 아래처럼 코드를 작성해 볼 수도 있을거 같네요
```suggestion
  const btnBgColor = () => {
    switch(type) {
        case("complete"):
            return ~
         case ("warning"):
             return ~
         default: 
             return ~ 
     }
  };
```

Reply:
오 수정하도록 하겠습니다.😄